src/components/navbar/NavBar.jsx

```diff
@@ -1,38 +1,82 @@
 import React from 'react';
-import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
-import AppBar from '@material-ui/core/AppBar';
+import {
+  Toolbar, Button, AppBar, Hidden,
+} from '@material-ui/core';
+import styled from 'styled-components';
+import NavDrawer from './navdrawer';
+import logo from '../../assets/socis_logo.png';
+
+const navLinks = [
+  { title: 'Clubs', path: '/clubs' },
+  { title: 'Policy', path: '/policy' },
+  { title: 'About', path: '/about' },
+  { title: 'Contact', path: '/contact' },
+];
 
 function NavBar() {
   return (
-    <div className="navbar">
-      <AppBar position="static" style={{ backgroundColor: '#660000' }}>
-        <Toolbar>
-          <Typography variant="h6">
-            <a href="/" style={styles.homeLink}>SOCIS</a>
-          </Typography>
-          <div className={styles.links}>
-            <Button color="inherit" href="/clubs">Clubs</Button>
-            <Button color="inherit" href="/policy">Policy</Button>
-            <Button color="inherit" href="/about">About</Button>
-            <Button color="inherit" href="/contact">Contact</Button>
-          </div>
-        </Toolbar>
-      </AppBar>
-    </div>
+    <StyledAppBar position="static" style={styles.appBar}>
+      <StyledToolbar>
+        <a href="/">
+          <Image src={logo} alt="SOCIS Logo" />
+        </a>
+        <WebView smDown>
+          <Links>
+            {navLinks.map(({ title, path }) => (
+              <Button key={title} color="inherit" href={path} style={styles.pageLink}>{title}</Button>
+            ))}
+          </Links>
+        </WebView>
+        <MobileView mdUp>
+          <NavDrawer navLinks={navLinks} />
+        </MobileView>
+      </StyledToolbar>
+    </StyledAppBar>
   );
 }
 
+const StyledAppBar = styled(AppBar)`
+  width: 100%;
+`;
+
+const StyledToolbar = styled(Toolbar)`
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between
+`;
+
+const Links = styled.div`
+`;
+
+const Image = styled.img`
+  margin-right: 0.5rem;
+  margin-top: 0.2rem;
+  margin-bottom: 0.2rem;
+  height: 4rem;
+`;
```

Comment:
Hb this instead?
```
const Image = styled.img`
  margin-top: 0.5rem;
  height: 4rem;
`;
```

Reply:
Original:
![image](https://user-images.githubusercontent.com/35578281/123245483-df446200-d4b2-11eb-8f06-766600400699.png)

Changed:
![image](https://user-images.githubusercontent.com/35578281/123245672-0ac74c80-d4b3-11eb-951d-a387838e78eb.png)

Worth the change?

Reply:
I think the original is more centered

Reply:
Yeah, I agree, looks better on other pages, was just looking at homepage